docs/topics_5/proc_web-obtaining-latest-image.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide_5/master.adoc
+
+
+[id="proc_web-obtaining-latest-image_{context}"]
+= Obtaining the latest image
+
+The first step in troubleshooting in an OpenShift Container Platform environment is to be sure you are using the latest image.
```

Comment:
This is not the first step if it does not apply to deployments with the template. What you change it to depends on what installations this advice applies to. I am not even sure how one would deploy MTA with an old image.